styles.css

```diff
@@ -447,3 +446,280 @@ input::placeholder {
 textarea::placeholder {
   color: #b3bac5;
 }
+
+/* To ignore for moblie */
+
+.desktop-header {
+  display: none;
+}
+
+.card-img1 {
+  display: none;
+}
+
+div.card-info1 {
+  display: none;
+}
+
+div.card-info2 {
+  display: none;
+}
+
+/* Desktop version */
+@media only screen and (min-width: 768px) {
+  .head-container {
+    margin: 0 155px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .desktop-header {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 10px 0 10px 10px;
+    gap: 32px;
+  }
+
+  .nav {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    text-decoration: none;
+  }
+
+  .nav a {
+    text-decoration: none;
+  }
+
+  .main-sec {
+    display: block;
+    padding-left: 34px;
+    margin-top: 50px;
+  }
+
+  .main {
+    display: block;
+    background-image: url(./images/Header\ bg@2x.png);
+    height: 90vh;
+    width: 100%;
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: contain;
+    padding-top: 1px;
+  }
```

Comment:
You have well implemented the hero section, however, there're a few things to point out for little adjustment in order to match the design template.

- The whole content of the hero section should be visible on height of 700px
- On the design, the text content has a little more width

Your design:
![image](https://user-images.githubusercontent.com/20554583/199763934-ee182330-8832-4aa7-bba3-e912ef10e1bf.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/199763909-0f8676e9-b16e-4ee7-9844-130cd71af75f.png)

Reply:
@emmiiorji Thanks for the feedback but if you pay close attention to the width on the screen shot of my website as compared to that on figma you will notice that mine is 1075px and that of figma is 1440px.  Try again please but this time with a with the same width and see the difference. thank you

Reply:
Hi @emmiiorji thanks

Reply:
Hi @shegy28,

On a second look, I noticed that it remains unresponsive and doesn't display as it should on proper desktop.

This seems to stem from the margin given to the wrapper class as shown:
![image](https://user-images.githubusercontent.com/20554583/199790533-5b6cc150-90bd-41a1-850c-8705f81a5074.png)


I recommend that you use a percentage value for the right and left margins e.g `margin: 200px 10%` displays properly on my device.

Reply:
@emmiiorji Now i am honestly confused and dont understand what you want me to do

Reply:
@emmiiorji  I have implemented the required changes to fit that of the aspect ratio of the template on figma which is 1440 X 916. Please i would appreciate if i am been reviewed that way thanks.

Reply:
@shegy28 I already marked this thread as deprecated, you may not worry.